src/wiki/templatetags/wiki_tags.py

```diff
@@ -12,11 +12,15 @@
 from django.utils.http import urlquote
 from django.utils.safestring import mark_safe
 from six.moves import filter
-from urllib.parse import urlencode
 from wiki import models
 from wiki.conf import settings
 from wiki.core.plugins import registry as plugin_registry
 
+try:
```

Comment:
Is this in six.moves? Otherwise should be moved to the `compat` module :)

Reply:
It is actually. Much nicer.